### PR TITLE
#2471 - BE | Enquire to study form: new field

### PR DESCRIPTION
**Pull request by William-Blackie**
Tickets: 

- https://projects.torchbox.com/projects/rca-django-cms-project/tickets/2471
- https://projects.torchbox.com/projects/rca-django-cms-project/tickets/2474

This pull request adds a new field `enquiry_questions` to the enquire to study form. This will then be sent to both QS and Mailchimp CRM.

Screenshot:

Screenshot 2023-03-09 at 10 46 55 AM

### CODE REVIEW

File: rca/enquire_to_study/views.py [hunk 1]
```diff
@@ -198,6 +199,9 @@ def post_qs(self, form_data):
             ],
         }
 
+        if enquiry_questions := form_data["enquiry_questions"]:
+            data["Notes"] += f"; Enquiry questions: {enquiry_questions}"
```
[the remark this answers is not in the document]
Comment (kevinhowbrook) [reply rca/enquire_to_study/views.py:203]: Also, I'm slightly concerned about the conditional on this. If there are no enquiry_questions we don't post anything, but QS might be expecting blank values but still expect the full post dict.

**Review by kevinhowbrook — CHANGES_REQUESTED** [review]
Looks fine to me, just one comment.

If there is time it might be worth updating the tests to include the new field https://github.com/torchbox/rca-wagtail-2019/blob/master/rca/enquire_to_study/tests/test_forms.py just to keep it up to date, looks like they don't fail as it's a non required field though.

File: rca/enquire_to_study/forms.py [hunk 2]
```diff
@@ -45,6 +45,13 @@ class EnquireToStudyForm(forms.Form):
         queryset=EnquiryReason.objects.all(), widget=forms.RadioSelect, empty_label=None
     )
 
+    enquiry_questions = forms.CharField(
+        label="Your questions (optional)",
+        help_text="If you have a specific enquiry or question, please include it here.",
+        required=False,
+        widget=forms.Textarea,
+    )
+
```
Comment (kevinhowbrook) [inline rca/enquire_to_study/forms.py:54]: Does this need adding to the export? https://github.com/torchbox/rca-wagtail-2019/blob/master/rca/enquire_to_study/wagtail_hooks.py#L34

Comment (patrickcuagan) [reply rca/enquire_to_study/forms.py:54]: Ah you're right I missed that one - I've updated this now. Thank you!

File: rca/enquire_to_study/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (patrickcuagan) [reply rca/enquire_to_study/views.py:203]: `Notes` is actually already in the dict so it'll still be in the dict even when there are no enquiry questions:

```
data = {
            ...,
            "Notes": f"Enquiry reason: {form_data['enquiry_reason'].reason}",
            ...,
        }
```

So the `;` was just to denote that the enquiry reason ended. What do you think?

[the remark this answers is not in the document]
Comment (kevinhowbrook) [reply rca/enquire_to_study/views.py:203]: Ah right - with you now, missed the `+=` there :rofl:

**Review by kevinhowbrook — APPROVED** [review]
Thanks @patrickcuagan

**Pull request merged**